### PR TITLE
Dt 1252 java to kotlin

### PR DESCRIPTION
**Pull request by dnyanesh-sonwane**
Converted Java classes to Kotlin

### CODE REVIEW

File: src/main/java/uk/gov/justice/digital/hmpps/oauth2server/resource/LoggingAccessDeniedHandler.kt [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import javax.servlet.http.HttpServletResponse
+
+@Component
+class LoggingAccessDeniedHandler : AccessDeniedHandler {
+
+  companion object {
+    private val log = LoggerFactory.getLogger(LoggingAccessDeniedHandler::class.java)
+  }
+
+  @Throws(IOException::class)
```
Comment (andymarke) [inline src/main/java/uk/gov/justice/digital/hmpps/oauth2server/resource/LoggingAccessDeniedHandler.kt:19]: I guess we could remove this annotation on a later PR?

File: src/test/java/uk/gov/justice/digital/hmpps/oauth2server/resource/VerifyMobileControllerTest.kt [hunk 2]
```diff
@@ -132,7 +129,6 @@ class VerifyMobileControllerTest {
   inner class MobileResend {
     @Test
     fun mobileResend_sendCode() {
-      whenever(userService.getUser(anyString())).thenReturn(User.builder().mobile("077009000000").build())
```
Comment (andymarke) [inline src/test/java/uk/gov/justice/digital/hmpps/oauth2server/resource/VerifyMobileControllerTest.kt:135]: How come these were no longer needed?

**Review by andymarke — DISMISSED** [review]
LGTM

File: src/test/java/uk/gov/justice/digital/hmpps/oauth2server/resource/VerifyMobileControllerTest.kt (hunk 2 above)
Comment (dnyanesh-sonwane) [reply src/test/java/uk/gov/justice/digital/hmpps/oauth2server/resource/VerifyMobileControllerTest.kt:135]: User service was unused in the class

File: src/main/java/uk/gov/justice/digital/hmpps/oauth2server/resource/LoggingAccessDeniedHandler.kt (hunk 1 above)
Comment (dnyanesh-sonwane) [reply src/main/java/uk/gov/justice/digital/hmpps/oauth2server/resource/LoggingAccessDeniedHandler.kt]: Removed now

**Review by andymarke — APPROVED** [review]
LGTM

**Pull request merged**